design/TF.md

```diff
@@ -1,79 +1,72 @@
+Tier IV's view is that control *is* affected by localization, even if trajectory following is done only in the odom frame. For example, if a trajectory is calculated from the shape of the lane specified in an HD map, the localization result will be indirectly used when projecting the trajectory into the odom frame, and thus the trajectory calculation will be thrown off if localization fails. Also, any map-based calculations that are done before trajectory calculation, such as shape optimization using the map's drivable areas or velocity optimization using the predicted trajectory of other vehicles (derived from lane shape information) will also be affected by localization failure. 
+
+In order to ensure that control is unaffected by localization failure, we require that all preceding calculations do not use the map->odom transform. However, since trajectory following comes after planning in Autoware, it is almost impossible to prevent map->odom from being involved in trajectory calculations. Although this might be possible if Autoware planned like a human (who only uses route graph information from the map and can obtain geometry information from perception), it is very unlikely that an autonomous driving stack is capable of ensuring safety without using geometry information. 
 
-Therefore, regardless of any frame that control is done, it will still have effects of localization results. To make control not to jerk or do sudden steering at localization failure, we set the following requirements to Localization module:
-* localizatoin result must be continuous
-* localization must detect localization failure and should not use the result to update vehicle pose.
+Therefore, regardless of the frame in which it is done, control will still be affected by localization. To control a vehicle without jerking or sudden steering in the event of localization failure, we set the following requirements for the Localization module:
+* Localization results must be continuous
+* Localization failures must be detected and the vehicle's pose should not be updated with any failed localization results
 
-Additionally, Localization architecture assumes sequential Bayesian Filter, such as EKF and particle filter, to be used to integrate twist and pose. Since it can also integrate odometry information, it can update TF at high frequency. 
-As a result, all the merits of odom->base_link stated in REP-105 can be satisfied by map->base_link, and thus there is no need of setting odom frame. 
+Additionally, the Localization architecture assumes that a sequential Bayesian Filter, such as EKF and particle filter, will be used to integrate twist and pose. Since it can also integrate odometry information, it can update TF at high frequency. As a result, all of the merits of odom->base_link stated in REP-105 can be satisfied by map->base_link, and thus there is no need to set the odom frame. 
```

Comment:
> such as EKF and particle filter
Are these examples of two separate Bayesian filters ("EKF filter", "particle filter"), or a single filter that is called "EKF and particle filter"?

> Since it can also integrate odometry information, it can update TF at high frequency
In this sentence, is 'it" referring to the same thing? If so, is "it" referring to a sequential Bayesian filter, or the Localization module?

Reply:
Since EKF stands for Extended Kalman Filter, I guess the original author of the document wanted to mention two separate Bayesian filters.